highfidelity/hifi
# Revert can cast shadow work

Reverts https://github.com/highfidelity/hifi/pull/12407 because it wasn't supposed to make it into the RC

Test plan:
1. Load the Paradise Island environment from the market
2. Verify there are no shadows
2. Select Menu->Settings->Developer->Menus
3. Select Developr->Graphics..
4. Verify both "Ambient occlusion" and "Shadows" combos show None.
5. Set "Shadows" to Enabled
6. Click "SAVE CHANGES" and verify that all objects in the scene have shadows.
